diff --git a/Makefile.am b/Makefile.am
index ebfd311..08ba0cc 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -7,7 +7,6 @@ BUILD_COMPONENTS += base
 if BUILD_DEVICE
 BUILD_COMPONENTS += libutils
 BUILD_COMPONENTS += logwrapper
-BUILD_COMPONENTS += libsync
 BUILD_COMPONENTS += fs_mgr
 BUILD_COMPONENTS += adb
 if USE_LOGD
diff --git a/configure.ac b/configure.ac
index 6b78f1e..fb56d71 100644
--- a/configure.ac
+++ b/configure.ac
@@ -112,8 +112,6 @@ AC_CONFIG_FILES([ \
     base/libbase.pc \
     logwrapper/Makefile \
     logwrapper/logwrapper.pc \
-    libsync/Makefile \
-    libsync/libsync.pc \
     fs_mgr/Makefile \
     fs_mgr/fs_mgr.pc \
     libsysutils/Makefile \
