From a89a1c8cb549713511d1fce63da178795f213b90 Mon Sep 17 00:00:00 2001
From: Erin Yan <xinyey@codeaurora.org>
Date: Thu, 24 Nov 2016 15:14:24 +0800
Subject: [PATCH] stream event extension

---
 src/pulse/def.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/pulse/def.h b/src/pulse/def.h
index db3230d..b3016e4 100644
--- a/src/pulse/def.h
+++ b/src/pulse/def.h
@@ -1028,6 +1028,9 @@ typedef void (*pa_free_cb_t)(void *p);
  * information, \since 0.9.15 */
 #define PA_STREAM_EVENT_REQUEST_UNCORK "request-uncork"
 
+/** A stream event notifying that client has lost the audio right */
+#define PA_STREAM_EVENT_RIGHT_LOST "right-lost"
+
 /** A stream event notifying that the stream is going to be
  * disconnected because the underlying sink changed and no longer
  * supports the format that was originally negotiated. Clients need
-- 
1.9.1

